src/scss/ScanContainer.scss

```diff
@@ -157,6 +157,11 @@ $lightgray: #d6d8dd;
           position: relative;
           bottom: 28px;
           text-transform: capitalize;
+
+          @media (max-width: 500px) {
+            left: -50vw;
```

Comment:
It still looks pretty jank but I understand what this patch is doing, after inspecting it carefully. A proper refactor would be better.